src/gui/wizard/owncloudadvancedsetuppage.cpp

```diff
@@ -196,9 +180,9 @@ void OwncloudAdvancedSetupPage::updateStatus()
         if (dirNotEmpty) {
             t += tr("<p><small><strong>Warning:</strong> The local folder is not empty. "
                     "Pick a resolution!</small></p>");
-            _ui.resolutionWidget->setVisible(dirNotEmpty);
+            _ui.resolutionStackedWidget->setCurrentIndex(1);
```

Comment:
Is there a specific advantage of using the index over the visibility? It seems a bit counterintuitive to me to do it like that. Please add a small comment that explains why this solution is better to avoid confusion in the future.

Reply:
If you hide and show a widget it messes with the layout, first its size is 0 than  not.
The stackedwidget always has a size and is properly layd out.